index.Rmd

```diff
@@ -1,105 +1,299 @@
+
+The formula is as below:
+[MNDWI=(Green-MIR)/ (Green+MIR)]
+MNDWI <- (image[[2]] - image[[3]])/(image[[2]] + image[[3]]).
+
+To plot the results with ggplot, we convert the raster into a data frame and use geom_tile.
+The plot generated by the codes below show where are located water/ water ice/ volatiles materials the most and we have then know where to land the rovers and start the mining process.
+
+Band4, Band5, and Band6 are use for Earth where Band4 represent the Green band in our case and Band6 represent the MIR band.
+
+For Mars,the formula change little bit because all bands are collected and stored in the same file in Mars Odysee website. False color or black and white could reveal the distribution of water. However, we use the formula:
+
+MNDWI <- (band4)/(1/band9).
+This formula is derived after multiple observations
+
+```{r}
+#For Earth data as example
+MNDWI <- (Band4 - Band6)/(Band4 + Band6)
+
+
+as(MNDWI, "SpatialPixelsDataFrame") %>% 
+  as.data.frame() %>%
+  ggplot(data = .) +
+  geom_tile(aes(x = x, y = y, fill = layer)) +
+  theme(axis.text = element_blank(),
+        axis.ticks = element_blank(),
+        panel.background = element_blank(),
+        panel.grid.minor = element_blank()) +
+  labs(title = "Water or Water ice or Volatiles materials", 
+       x = " ", 
+       y = " ") +
+  scale_fill_gradient(high = "#CEE50E", 
+                      low = "#087F28",
+                      name = "Water or Water ice or Volatiles materials")
+
+#For Mars. 
+#We derived This formula is derived after multiple observations and applied to different data before using it.
+MNDWI <- (band4)/(1/band9)
+#MNDWI <- (band4-band9)/(band4+band9)
+
+
+as(MNDWI, "SpatialPixelsDataFrame") %>% 
+  as.data.frame() %>%
+  ggplot(data = .) +
+  geom_tile(aes(x = x, y = y, fill = layer)) +
+  theme(axis.text = element_blank(),
+        axis.ticks = element_blank(),
+        panel.background = element_blank(),
+        panel.grid.minor = element_blank()) +
+  labs(title = "Water or Water ice or Volatiles materials", 
+       x = " ", 
+       y = " ") +
+  scale_fill_gradient(high = "#CEE50E", 
+                      low = "#087F28",
+                      name = "Water or Water ice or Volatiles materials")
 ```
 
+## PART2: Visualization of landing/mining sites on map
+
+I loaded required packages
+
+```{r, message=F, warning=F}
+library(threejs)
+```
+
+##Results2
+I firstly prepared and loaded the data and then with the package called "streejs" I plot the exact locations of landing and mining sites on the Martian global map. I created a cvs file where I stored the latitudes and logitude of the future landing and mining sites selested and call that file in R.
+
+
+```{r}
+library(threejs)
+library(csvread)
+the_martian <- read.csv("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/the_martian.csv", comment.char="#")
```

Comment:
Where did these come from? They are not connected to the data above?  Also, this file will need to be included in the repository or publically available.

Reply:
In the introduction you said you were going to identify the best sites using the index - but that's not what is shown here.  Please update either the introduction or the methods.